src/extension/log/OutputChannelLogger.ts

```diff
@@ -63,7 +66,21 @@ export class OutputChannelLogger implements ILogger {
         }
 
         if (level >= LogHelper.LOG_LEVEL) {
-            message = OutputChannelLogger.getFormattedMessage(message, level);
+            message = OutputChannelLogger.getFormattedMessage(message, LogLevel[level], this.logTimestamps);
+            this.channel.appendLine(message);
+            if (this.channelLogFileStream) {
+                this.channelLogFileStream.write(message);
+            }
+        }
+    }
+
+    public logWithTag(tag: string, message: string, level: LogLevel): void {
+        if (LogHelper.LOG_LEVEL === LogLevel.None) {
+            return;
+        }
+
+        if (level === LogLevel.Tag) {
```

Comment:
Do we need this check? It is an independent function that can work without bindings to `LogLevel`.